app/templates/work-experience-Juli.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
+
+<div class= "Juli's Work Experience Page">
+    <h1> Juli's Work Experience Page</h1>
+    <h2> Full Stack Developer at Stocate</h2>
```

Comment:
im just a volunteer here, so it is more accurate to write 

Full Stack Developer Volunteer